content/en/monitors/guide/template-variable-evaluation.md

```diff
@@ -0,0 +1,98 @@
+| cosh(var)           | Returns the hyperbolic cosine of var                                                                                                       |
+| acos(var)           | Returns the arc cosine of var, in radians                                                                                                  |
+| acosh(var)          | Returns the inverse hyperbolic cosine of var                                                                                               |
+| tan(var)            | Returns the tangent of var radians                                                                                                         |
+| tanh(var)           | Returns the hyperbolic tangent of var                                                                                                      |
+| atan(var)           | Returns the arc tangent of var, in radians                                                                                                 |
+| atan2(var1, var2)   | Returns atan(var1 / var2), in radians                                                                                                      |
+| atanh(var)          | Returns the inverse hyperbolic tangent of var                                                                                              |
+| exp(var)            | Returns e (the base of the natural logarithm) raised to the power of var                                                                   |
+| log10(var)          | Returns the base-10 logarithm of var                                                                                                       |
+| abs(var)            | Returns the absolute value of var                                                                                                          |
+| int(var)            | Returns var as an integer                                                                                                                  |
+| float(var)          | Returns var as a float                                                                                                                     |
+| trunc(var)          | Returns var as an integer                                                                                                                  |
+| round(var)          | Returns var rounded to the nearest integer                                                                                                 |
+| ceil(var)           | Returns the ceiling of var (the smallest integer that is greater than or equal to var)                                                     |
+| floor(var)          | Returns the floor of var (the largest integer that is less than or equal to var)                                                           |
+| dec(var)            | Returns the numbers to the right of the decimal point, for example 12.345 returns 0.345.                                                   |
```

Comment:
**Change this sentence to:** Returns the numbers to the right of the decimal point, for example if var = 12.345 then dec(var) returns 0.345